code/xcvm/lib/core/src/bridge.rs

```diff
@@ -0,0 +1,73 @@
+use codec::{Decode, Encode};
+use scale_info::TypeInfo;
+use serde::{Deserialize, Serialize};
+
+/// Newtype for XCVM networks ID. Must be unique for each network and must never change.
+/// This ID is an opaque, arbitrary type from the XCVM protocol and no assumption must be made on
+/// how it is computed.
+#[cfg_attr(feature = "std", derive(schemars::JsonSchema))]
+#[derive(
+	Copy,
+	Clone,
+	PartialEq,
+	Eq,
+	PartialOrd,
+	Ord,
+	Debug,
+	Encode,
+	Decode,
+	TypeInfo,
+	Serialize,
+	Deserialize,
+)]
+#[repr(i32)]
+pub enum BridgeSecurity {
+	None,
```

Comment:
Please remove this variant as its isomorphic to `Option<BridgeSecurity>`

Reply:
It is more convenient to have None variant here though because it makes it super convenient to compare bridge securities since they can be represented as i32